src/script.js

```diff
@@ -27,13 +27,16 @@ var backUrl = localStorage.getItem("theValue");
 var box = document.getElementById("box");
 
 window.onload = function () {
-    box.style.backgroundImage = "url(" + backUrl + ")";
-    box.style.backgroundRepeat = "no-repeat";
-    box.style.backgroundSize = "cover";
-    headerbox.style.backgroundColor = theme;
-    mic.style.color = theme;
-    but.style.color = theme;
-    console.log(theme);
+
+    if(backUrl) {
+        box.style.backgroundImage = "url(" + backUrl + ")";
+        box.style.backgroundRepeat = "no-repeat";
+        box.style.backgroundSize = "cover";
+        headerbox.style.backgroundColor = theme;
```

Comment:
@ms10398 ,shouldn't lines 35,36,37,38  be in a different check?

Reply:
I didnt got it?
What do you mean exactly